src/reducers/index.js

```diff
@@ -0,0 +1,19 @@
+import { combineReducers } from 'redux';
+
+// TODO: remove as soon as the first useful reducer is added
+const dummy = (state = 0, action) => {
+  if(action.type === 'INC') {
+    return state += 1;
+  }
+  else if(action.type === 'THUNK_INC') {
+    return state += 11;
+  }
+  return state;
+};
+
+
+const reducers = combineReducers({
+  dummy
+});
+
+export default reducers;
```

Comment:
I guess, these doesn't hurt at the moment, but don't see obvious reason to commit them either.